.gitignore

```diff
@@ -367,6 +367,7 @@ MigrationBackup/
 **/*.processed.yaml
 **/*.output.ipynb
 **/*.processed.ipynb
+**/*.yaml.updated
```

Comment:
why is this added ?

Reply:
Because these files get generated when you run tests locally and they shouldn't be checked in 🙂